x/lockup/keeper/lock.go

```diff
@@ -65,42 +79,89 @@ func (k Keeper) GetModuleBalance(ctx sdk.Context) sdk.Coins {
 
 // GetModuleLockedCoins Returns locked balance of the module
 func (k Keeper) GetModuleLockedCoins(ctx sdk.Context) sdk.Coins {
-	return k.getCoinsFromIterator(ctx, k.LockIteratorAfterTime(ctx, ctx.BlockTime()))
+	// all not unlocking + not finished unlocking
+	notUnlockingCoins := k.getCoinsFromIterator(ctx, k.LockIterator(ctx, types.KeyPrefixNotUnlocking))
+	unlockingCoins := k.getCoinsFromIterator(ctx, k.LockIteratorAfterTime(ctx, types.KeyPrefixUnlocking, ctx.BlockTime()))
+	return notUnlockingCoins.Add(unlockingCoins...)
 }
 
 // GetAccountUnlockableCoins Returns whole unlockable coins which are not withdrawn yet
 func (k Keeper) GetAccountUnlockableCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins {
-	return k.getCoinsFromIterator(ctx, k.AccountLockIteratorBeforeTime(ctx, addr, ctx.BlockTime()))
+	return k.getCoinsFromIterator(ctx, k.AccountLockIteratorBeforeTime(ctx, types.KeyPrefixUnlocking, addr, ctx.BlockTime()))
 }
 
```

Comment:
Let's have a `GetAccountUnlockingCoins`

Reply:
UnlockingCoins should include unlockable coins as well or only unlocking started but not unlockable coins?

Reply:
The second one:  `only unlocking started but not unlockable coins`